app/assets/javascripts/Components/submission_table.jsx

```diff
@@ -348,12 +350,70 @@ class RawSubmissionTable extends React.Component {
 
         received: data => {
           // Called when there's incoming data on the websocket for this channel
-          this.fetchData();
+          if (data["status"] != null) {
+            let message_data = this.generateFlashMessageContentsUsingStatus(data);
+            this.renderFlashMessages(message_data);
+          }
+          if (data["update_table"] != null) {
+            this.fetchData();
+          }
         },
       }
     );
   };
 
+  generateFlashMessageContentsUsingStatus = status_data => {
+    let message_data = {};
+
+    switch (status_data["status"]) {
+      case "failed":
+        if (!status_data["exception"] || !status_data["exception"]["message"]) {
+          message_data["error"] = "Failed";
```

Comment:
Remember that all user-facing strings should be internationalized. I suspect that the previous statuses (from `activejob-status`) actually weren't, but we can do better now (especially since I never liked those terse one-word statuses anyway). Please use the following:

- `failed`, no message: `This job failed due to an unexpected error.`
- `failed`, message: `This job failed due to the following error:` (plus the error message, of course)
- `completed`: `This job is now complete.`
- `queued`: `This job is waiting to be run.`

Note that the in-progress message is fine, but should also be internationalized.